api/src/test/java/com/ford/labs/retroquest/deprecated_tests/ColumnCombinerServiceTest.java

```diff
@@ -159,30 +159,24 @@ void columnTitleInOrder() {
         assertThat(response.getColumns().get(3).getTitle()).isEqualTo("Action Item");
     }
 
-    // "should put active items in correct list"
+    // "should put all active and completed items in one list"
```

Comment:
Is there a reason this isn't just the test name?

Reply:
This should be a separate PR, but we should just move all the tests in deprecated_tests back into the main test module and fix all the naming inconsistencies.